sqlite3.go

```diff
@@ -1522,10 +1522,10 @@ func (d *SQLiteDriver) Open(dsn string) (driver.Conn, error) {
 		// Before going any further, we need to check that the user
 		// has provided an username and password within the DSN.
 		// We are not allowed to continue.
-		if len(authUser) < 0 {
+		if len(authUser) <= 0 {
```

Comment:
Oh, why I did miss this wrong code... Thanks.

I prefer ` == 0`.

Reply:
Done